### ISSUE TITLE
Can't play Metal Gear using a gamepad, buttons aren't assigned

### ISSUE THREAD

**Issue by sergiobenrocha2** [libretro/blueMSX-libretro#16]
It's not possible to play Metal Gear games, some buttons aren't assigned to the retropad, like F1 to F5.

It should be like [fMSX](https://github.com/libretro/fmsx-libretro/blob/master/libretro.c#L101)

There's an workaround if you use keyboard: change User 1 Device Type to RetroKeyboard and choose a hotkey under Input Hotkey Binds --> Enable hotkey

**Comment by gingerbeardman** [comment]
@sergiobenrocha2 can you describe your workaround a little more? I'm trying to play a game that requires me to select a difficulty before I begin, by pressing keys 1, 2 or 3. Is this possible using your workaround? Thanks

**Comment by splinefx** [comment]
@sergiobenrocha2, @gingerbeardman 
Is there any update on this? I'm facing the exact same problem right now, using RetroArch.

**Comment by gingerbeardman** [comment]
@splinefx did you try the workaround in the OP?

**Comment by splinefx** [comment]
@gingerbeardman yes, it worked. I didn't figure out how this thing sets up bc it's a bit inconsistent (usually you have the action on the left, and gamepad button assigned to it on the right, but here you have gamepad button on the left instead of an action, and keyboard button assigned to it on the right). But I finally made it work.
